### PR TITLE
Align with Governance audit

### LINKED ISSUE #494: New Governance Metric - OSC Governance/Security TWG
Per the Security TWG: Please update/add Security.md

Please refer to: https://github.com/IntersectMBO/Open-Source-Office/blob/main/SECURITY.md

The initial deadline for these changes to be made is 30 September 2024. Thank you.

Ref: Issue #482

### PR DESCRIPTION
**Pull request by Ranchhand87**
# Description

Fixes #494, Addresses requests in #482. Please review all updates. If this is to be merged with no changes, please enable Discussions or contact @Ranchhand87 to enable.  

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages.
      **_New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-base/blob/master/RELEASING.md#changelogmd))
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/intersectmbo/cardano-base/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated.
      **_If you change the bounds in a cabal file, that package itself must have a version increase._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-base/blob/master/RELEASING.md#versioning-process))
- [x] Self-reviewed the diff

### CODE REVIEW

File: .github/ISSUE TEMPLATES/config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Contributing guidelines
+    url: https://github.com/cardano-base/blob/master/CONTRIBUTING.md
```
Comment (lehins) [inline .github/ISSUE TEMPLATES/config.yml:4]:
```suggestion
    url: https://github.com/IntersectMBO/cardano-base/blob/master/CONTRIBUTING.md
```

File: .github/ISSUE TEMPLATES/config.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    about: Some rules & processes we honor.
+
+  - name: Feature ideas
+    url: https://github.com/cardano-base/discussions/categories/ideas
+    about: Maybe someone else had the same or a similar idea already?
+
+  - name: All issues
+    url: https://github.com/cardano-base/issues
+    about: Check whether your issue is not already covered here.
```
Comment (lehins) [inline .github/ISSUE TEMPLATES/config.yml:13]: File should end with a newline
```suggestion
    about: Check whether your issue is not already covered here.

```

File: .github/ISSUE TEMPLATES/config.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Contributing guidelines
+    url: https://github.com/cardano-base/blob/master/CONTRIBUTING.md
+    about: Some rules & processes we honor.
+
+  - name: Feature ideas
+    url: https://github.com/cardano-base/discussions/categories/ideas
+    about: Maybe someone else had the same or a similar idea already?
+
```
Comment (lehins) [inline .github/ISSUE TEMPLATES/config.yml:10]: There is no need for discussions in cardano-base. Opening an issue for any reason is totally fine.
```suggestion
```

File: .github/ISSUE TEMPLATES/feature_idea_base.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  - type: textarea
+    id: how
+    attributes:
+      label: How
+      description: How could we realize this feature
+      placeholder: |
+        Which technical solutions, libraries or systems should be used, which
+        components need to change, steps how to implement this, ...
+    validations:
+      required: true
```
Comment (lehins) [inline .github/ISSUE TEMPLATES/feature_idea_base.yml:43]: I copied the template from Ledger with some adjustments
```suggestion
name: Cardano Base Repository Issue
description: Issue, Feature Enhancement, or Fix for the cardano-base repository
### Description
A clear and concise description of the problem or feature request. Include relevant context, details and the package name(s) that is affected

### Steps to Reproduce (for bugs)
1. Step 1
2. Step 2
3. Step 3
(Include screenshots if applicable)

### Expected Behavior
A clear description of what you expected to happen.

### Actual Behavior
What actually happens when following the steps. Include any error messages.

### Environment (if applicable)
- Package Name and Version:
- Operating System:
- Nix/Cabal Version:

### Additional Context
Add any other context about the problem or request here. Include possible solutions, discussions, or relevant documentation links.

### Labels
(For maintainers: add relevant labels such as `bug`, `enhancement`, `technical-debt`, etc.)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- [cardano-crypto-class](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-class/CHANGELOG.md) 
+- [cardano-crypto-praos](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-praos/CHANGELOG.md) 
+- [cardano-crypto-tests](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-tests/CHANGELOG.md)  
+- [cardano-mempool](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-mempool/CHANGELOG.md) 
+- [cardano-slotting](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-slotting/CHANGELOG.md) 
+- [cardano-strict-containers](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-strict-containers/CHANGELOG.md) 
+- [heapwords](https://github.com/IntersectMBO/cardano-base/blob/master/heapwords/CHANGELOG.md) 
+- [measures](https://github.com/IntersectMBO/cardano-base/blob/master/measures/CHANGELOG.md) 
+- [orphans-deriving-via](https://github.com/IntersectMBO/cardano-base/blob/master/orphans-deriving-via/CHANGELOG.md)
+- [strict-checked-vars](https://github.com/IntersectMBO/cardano-base/blob/master/strict-checked-vars/CHANGELOG.md)
```
Comment (lehins) [inline CHANGELOG.md:16]: This package is being removed. See #505 
```suggestion
```

File: .github/ISSUE TEMPLATES/config.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  - name: All issues
+    url: https://github.com/cardano-base/issues
```
Comment (lehins) [inline .github/ISSUE TEMPLATES/config.yml:12]:
```suggestion
    url: https://github.com/IntersectMBO/cardano-base/issues
```

**Review by lehins — CHANGES_REQUESTED** [review]
Some of the same changes as were done for cardano-ledger are needed

**Comment by Ranchhand87** [comment]
Committed the suggestions. Apologies for any extra work, I had stepped back for a few days with other tasks, but coming back to ensure I am double and triple checking future commits. TIA

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+- [base-deriving-via](https://github.com/IntersectMBO/cardano-base/blob/master/base-deriving-via/CHANGELOG.md)  
+- [cardano-binary](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-binary/CHANGELOG.md) 
+- [cardano-crypto-class](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-class/CHANGELOG.md) 
+- [cardano-crypto-praos](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-praos/CHANGELOG.md) 
+- [cardano-crypto-tests](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-tests/CHANGELOG.md)  
+- [cardano-mempool](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-mempool/CHANGELOG.md) 
+- [cardano-slotting](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-slotting/CHANGELOG.md) 
+- [cardano-strict-containers](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-strict-containers/CHANGELOG.md) 
+- [heapwords](https://github.com/IntersectMBO/cardano-base/blob/master/heapwords/CHANGELOG.md) 
+- [measures](https://github.com/IntersectMBO/cardano-base/blob/master/measures/CHANGELOG.md)
```
Comment (lehins) [inline CHANGELOG.md:14]: We don't allow trailing whitespaces in markdown files
```suggestion
- [base-deriving-via](https://github.com/IntersectMBO/cardano-base/blob/master/base-deriving-via/CHANGELOG.md)
- [cardano-binary](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-binary/CHANGELOG.md)
- [cardano-crypto-class](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-class/CHANGELOG.md)
- [cardano-crypto-praos](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-praos/CHANGELOG.md)
- [cardano-crypto-tests](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-crypto-tests/CHANGELOG.md)
- [cardano-mempool](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-mempool/CHANGELOG.md)
- [cardano-slotting](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-slotting/CHANGELOG.md)
- [cardano-strict-containers](https://github.com/IntersectMBO/cardano-base/blob/master/cardano-strict-containers/CHANGELOG.md)
- [heapwords](https://github.com/IntersectMBO/cardano-base/blob/master/heapwords/CHANGELOG.md)
- [measures](https://github.com/IntersectMBO/cardano-base/blob/master/measures/CHANGELOG.md)
```

**Review by lehins — APPROVED** [review]
Looks great. Thank you!

**Pull request merged**